components/sidebar/sidebar_service.cc

```diff
@@ -48,35 +48,50 @@ SidebarItem GetBuiltInItemForType(SidebarItem::BuiltInItemType type) {
                                      IDS_SIDEBAR_BRAVE_TALK_ITEM_TITLE),
                                  SidebarItem::Type::kTypeBuiltIn,
                                  SidebarItem::BuiltInItemType::kBraveTalk,
-                                 false);
+                                 /* open_in_panel = */ false);
     case SidebarItem::BuiltInItemType::kWallet:
       return SidebarItem::Create(GURL("chrome://wallet/"),
                                  brave_l10n::GetLocalizedResourceUTF16String(
                                      IDS_SIDEBAR_WALLET_ITEM_TITLE),
                                  SidebarItem::Type::kTypeBuiltIn,
-                                 SidebarItem::BuiltInItemType::kWallet, false);
+                                 SidebarItem::BuiltInItemType::kWallet,
+                                 /* open_in_panel = */ false);
     case SidebarItem::BuiltInItemType::kBookmarks:
       return SidebarItem::Create(brave_l10n::GetLocalizedResourceUTF16String(
                                      IDS_SIDEBAR_BOOKMARKS_ITEM_TITLE),
                                  SidebarItem::Type::kTypeBuiltIn,
                                  SidebarItem::BuiltInItemType::kBookmarks,
-                                 true);
+                                 /* open_in_panel = */ true);
     case SidebarItem::BuiltInItemType::kReadingList:
       return SidebarItem::Create(
           // TODO(petemill): Have these items created under brave/browser
           // so that we can access common strings, like IDS_READ_LATER_TITLE.
           brave_l10n::GetLocalizedResourceUTF16String(
               IDS_SIDEBAR_READING_LIST_ITEM_TITLE),
           SidebarItem::Type::kTypeBuiltIn,
-          SidebarItem::BuiltInItemType::kReadingList, true);
-    case SidebarItem::BuiltInItemType::kHistory:
-      return SidebarItem::Create(GURL("chrome://history/"),
-                                 brave_l10n::GetLocalizedResourceUTF16String(
-                                     IDS_SIDEBAR_HISTORY_ITEM_TITLE),
-                                 SidebarItem::Type::kTypeBuiltIn,
-                                 SidebarItem::BuiltInItemType::kHistory, true);
-    default:
+          SidebarItem::BuiltInItemType::kReadingList,
+          /* open_in_panel = */ true);
+    case SidebarItem::BuiltInItemType::kHistory: {
+      // TODO(sko) When should we show history item?
```

Comment:
Maybe we'll have another history page that fits for sidebar in the future.

Reply:
Yeah, It seems that upstream folks are implementing this too.